wa-admin/src/app/pages/home/components/add-user/add-user.component.ts

```diff
@@ -8,7 +8,7 @@ import { IInvitationRecord } from 'src/app/shared/interfaces/invitation-record.i
 import { AlertService } from 'src/app/services/alert.service';
 import { Router } from '@angular/router';
 
-const url = environment.publicUrl;
```

Comment:
I will work on parametrizing this further, however due to time constraints we may want to try this option in OpenShift first so we can test the app flow in the cloud environment